### DISCUSSION TITLE
Does anyone else get their auth session logged out in the Pixie Plus app after authing in Home Assistant?

### DISCUSSION THREAD

**Discussion by aszorkov** [nirnachmani/Pixie-Plus-for-Home-Assistant#12] [Q&A] [answered]
I'm thinking that maybe one of the ids is unique to a user or something and so will log out other sessions when used on another device?

After authenticating in the HA integration my session in the app is logged out, and logging back in to Pixie Plus app will log out the integration.

I should be able to get around this by generating a new user just for the HA integration, but just wondering if I'm the only one who experiences this as I didn't see it mentioned anywhere

**Comment by nirnachmani** [comment, marked as answer]
This is indeed the case. The app produces one of those ids that is unique for the device. You can login from an old phone or a virtual device to create a new one so you are not using the one in your main app.

**Reply by aszorkov** [reply]
Ah of course, it's tied to the app instance. Cheers!

**Comment by smashtup** [comment]
Apologies for the late response.

Can confirm installation-id value can be configured by generating and copying a new UUID. ie. https://www.uuidgenerator.net/guid
